app/views/posts/index.html.erb

```diff
@@ -1,2 +1,12 @@
 <h1>Posts page </h1>
-<p> List of users posts here </p>
\ No newline at end of file
+<p> List of users posts here </p>
+<% @posts.each do |post|  %>
```

Comment:
- [ ] According to the project requirement you should  uses `post.recent_comments` to render 5 most recent comments . Please fix it in order to make your app work as expected :+1: